convert.js

```diff
@@ -50,23 +50,10 @@ function getConvertedArray(arr) {
   return arr.map((obj) => convert(obj));
 }
 
-function toCamelCase(key) {
```

Comment:
The old toCamelCase function did not pass 2 of the 3 tests. I remade it in the way I would've solved it.
You don't have to use my solution, it is your choice :) 
I just want to give inspiration 😄

Reply:
@SimpleCookie that's great! the code is pretty neat! Thank you for your effort to contribute the testing and the implementation :)

Reply:
You are welcome :)